tests/v2/actions/secrets/secrets.go

```diff
@@ -0,0 +1,52 @@
+package secrets
+
+import (
+	"fmt"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/extensions/secrets"
+	namegen "github.com/rancher/shepherd/pkg/namegenerator"
+	"github.com/rancher/shepherd/pkg/wrangler"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// CreateSecret is a helper to create a secret
+func CreateSecret(client *rancher.Client, clusterID, namespaceName string) (*corev1.Secret, error) {
+	var ctx *wrangler.Context
+	var err error
+
+	if clusterID != "local" {
+		ctx, err = client.WranglerContext.DownStreamClusterWranglerContext(clusterID)
+		if err != nil {
+			return nil, fmt.Errorf("failed to get downstream context: %w", err)
+		}
+	} else {
+		ctx = client.WranglerContext
+	}
+
+	secretName := namegen.AppendRandomString("testsecret")
+	secretTemplate := secrets.NewSecretTemplate(secretName, namespaceName, map[string][]byte{
+		"hello": []byte("world"),
```

Comment:
Instead of hard coding here, can we make the function accept data and send in the info in that var.

Reply:
Updated the function to accept data as input instead of hard coding it